*** sccp_pbx.c	2009-03-16 22:09:48.000000000 +0300
--- ../sccp_pbx.c	2010-03-12 19:57:48.062027617 +0300
*************** static char *sccp_control2str(int state)
*** 584,590 ****
  		case AST_CONTROL_VIDUPDATE:
  				return "VideoFrameUpdate";
  #endif
! #ifdef CS_AST_CONTROL_T38
  		case AST_CONTROL_T38:
  				return "T38RequestNotification";
  #endif
--- 584,590 ----
  		case AST_CONTROL_VIDUPDATE:
  				return "VideoFrameUpdate";
  #endif
! #ifdef CS_AST_CONTROL_T38_PARAMETERS
  		case AST_CONTROL_T38:
  				return "T38RequestNotification";
  #endif
